### ISSUE TITLE
language-thrift being taken out of stackage-nightly

### ISSUE THREAD

**Issue by cdornan** [abhinav/language-thrift#13]
We are taking language-thrift out of stackage nightly because it has been blocking megaparsec-6 for 4 weeks. @abhinav can you get laguage-thrift to work with megaparsec-6? We would love to have it back in nightly.

**Comment by abhinav** [comment]
Hey, thanks for letting me know. I won't have time to fix this anytime soon so I'm okay with missing the next LTS train.

**Comment by cdornan** [comment]
I entirely understand -- are you expecting to have to do anything beyond adjusting the upper bounds for megaparsec?

**Comment by abhinav** [comment]
Yeah, a bunch of the APIs changed in the new release. I started working on it (#11) when I saw the original Stackage issue but haven't had time to see it through just yet.
